roles/telemetry_verify_metrics/tasks/verify_mariadb_metrics.yml

```
@@ -0,0 +1,48 @@
+- name: Verify mariadb scrapeconfig exists
+  ansible.builtin.include_role:
+    name: common
+  vars:
+    common_cr_test_id: RHOSO-xxxx
+    common_cr_list:
+      - kind: scrapeconfigs.monitoring.rhobs
+        name: telemetry-mysqld-exporter
+
+- name: Get galera CR names
+  ansible.builtin.shell: |
+    oc get galeras -o custom-columns=NAME:.metadata.name --no-headers
+  register: cr_names
+  changed_when: false
+  failed_when: cr_names.rc >= 1 or cr_names.stdout == ""
+
+- name: Create mariadb pod list
+  ansible.builtin.set_fact:
+    common_pod_list: "{{ common_pod_list | default([]) + [item + '-galera-0' + '-mysqld-exporter-0'] }}"
+  loop: "{{ cr_names.stdout_lines }}"
+
+- name: Verify galera pods are running
+  ansible.builtin.include_role:
+    name: common
+  vars:
+    common_pod_test_id: RHOSO-xxxx
+    common_pod_status_str: "Running"
+    common_pod_nspace: openstack
```

```suggestion
- name: Create galera pod list
  ansible.builtin.set_fact:
    galera_pod_list: "{{ [item + '-galera-0'] }}"
  loop: "{{ cr_names.stdout_lines }}"

- name: Verify galera and mysqld-exporter pods are running
  ansible.builtin.include_role:
    name: common
  vars:
    common_pod_test_id: RHOSO-xxxx
    common_pod_status_str: "Running"
    common_pod_nspace: openstack
    common_pod_list: galera_pod_list + ['mysqld-exporter-0']
```
The original ansible creates names, which don't make much sense like "openstack-cell1-galera-0-mysqld-exporter-0". But instead you need a list of ["openstack-cell1-galera-0", "openstack-galera-0", "mysqld-exporter-0"]. I think the suggestion should achieve that.